src/Drivers/Zibal/Zibal.php

```diff
@@ -84,13 +84,21 @@ public function purchase()
             $description = $this->settings->description;
         }
 
+        $allowedCards = null;
+        if (!empty($details['allowedCards'])) {
+            $allowedCards = $details['allowe$allowedCards'];
```

Comment:
is this line correct ? i mean `$details['allowe$allowedCards'];`

Reply:
No it's a typo. It must have been $details['allowedCards']; instead. I will fix and rebase.